### PR TITLE
fix: readme typo >> clone through ssh error

### PR DESCRIPTION
**Pull request by hgp22**
Most useful pull request.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -76,7 +76,7 @@ Help is welcome!
 ** Building
 
 #+begin_example
-$ git clone git://github.com/djcb/mu.git
+$ git clone git@github.com:djcb/mu.git
```
Comment (djcb) [inline README.org:79]: Think it should be `https://github.com/djcb/mu.git`, or? Fewer things can go wrong that way

Comment (hgp22) [reply README.org:79]: I tried to clone the repo, copy & paste from the readme, and it didnt work.
This was the input/output:
```
❯ git clone git://github.com/djcb/mu.git
Cloning into 'mu'...
fatal: unable to connect to github.com:
github.com[0: 140.82.121.4]: errno=Connection timed out
```
I just tried it again and gave me the same error.

Comment (djcb) [reply README.org:79]: Yeah, `git://` won't work but `https://`:
```
git clone https://github.com/djcb/mu.git
Cloning into 'mu'...
remote: Enumerating objects: 37259, done.
```

Comment (hgp22) [reply README.org:79]: The link on the readme was not working.
I prefer ssh, but whatever works.
```suggestion
$ git clone https://github.com/djcb/mu.git
```

**Comment by djcb** [comment]
Thanks! Merged. Yeah, prefer ssh as well, but https is more likely to work.

**Pull request merged**